src/DIRAC/MonitoringSystem/Client/MonitoringReporter.py

```diff
@@ -49,6 +49,7 @@ def __init__(self, monitoringType="", failoverQueueName="dirac.monitoring"):
         self.__monitoringType = monitoringType
         self.__failoverQueueName = failoverQueueName
         self.__defaultMQProducer = None
+        self.__commitTimer = threading.Timer(5, self.commit)
```

Comment:
This does not seem used.

Reply:
You are right. Forgot it there when I wanted to implement the delayedCommit() method,